apollo-portal/src/main/scripts/startup.sh

```diff
@@ -63,6 +67,22 @@ function checkPidAlive {
     exit 1;
 }
 
+function existProcessUsePort() {
+    if [ "$(curl -X GET --silent --connect-timeout 1 --max-time 2 --head $SERVER_URL | grep "HTTP")" != "" ]; then
+        true
+    else
+        false
+    fi
+}
+
+function isServiceRunning() {
+    if [ "$(curl -X GET --silent --connect-timeout 1 --max-time 2 $SERVER_URL/signin | grep "config.png")" != "" ]; then
```

Comment:
this line is differenct with configservice and adminservice, which use `$SERVER_URL/apollo/version`

Reply:
How about `/health`?

Reply:
`/health` is better